terraform/s3/bind/data.tf

```diff
@@ -74,6 +74,22 @@ data "aws_iam_policy_document" "user_policy" {
       format("%s/*", var.arn)
     ]
   }
+
+  dynamic "statement" {
+    for_each = var.allowed_aws_vpc_id != "" ? [1] : []
```

Comment:
Is this better than using `count` which is commonly used in other `*.tf` files? I'm not asking for it to be changed, just curious.

Reply:
A `dynamic` block acts much like a for expression and the `for_each` argument provides the complex value to iterate over. This is a build-in expression that does not admit the `count`.

Reply:
Thanks. I missed that it was a `dynamic`.